docs/requests.md

```diff
@@ -59,5 +59,15 @@ Under current campus access policies if you are interactively testing this gem y
 2. Enter the rails console for environment you want to working with. ```RAILS_ENV=production bundle exec rails c```
 3. Add your net ID: ```CampusAccess.create(uid:'mynetid')```
 
+## Getting a Count of Requested Clancy Items
 
+1. Start the Rails console in one of the catalog web servers
+```
+rails c
+```
 
+2. Example Commands
+```
+item = Requests::ClancyItem.new()
+item.send(:get_clancy, url: "retrievedlist/v1/20220101/20220401/MQ")
```

Comment:
@rladdusaw can you include how to cancel the request from this comment https://github.com/pulibrary/orangelight/pull/2913#issuecomment-1087690513

Reply:
Done.